### PR TITLE
#1590 - using current_release.txt to read release artifact URL programmatically

### PR DESCRIPTION
**Pull request by smijar**
Please see #1590 for more details.

### CODE REVIEW

**Review by johnmuth81 — APPROVED** [review]
lgtm

**Review by gnode1 — APPROVED** [review]
LGTM after you confirm changes in `payloads/nginx/nginx/.gitignore`

File: .gitignore [hunk 1]
```diff
@@ -27,8 +27,10 @@ payloads/nginx/nginx
 cscope.out
 **/cscope.out
 .vscode/tags
+.vscode/*.log
```
[the remark this answers is not in the document]
Comment (smijar) [reply .gitignore:30]: Can I add .vscode/** to .gitignore?  Some extension in my visual studio code keeps generating .log files in the folder and they get included in git changes to the repo.

[the remark this answers is not in the document]
Comment (smijar) [reply .gitignore:30]: I can remove the cscope stuff out.  Have no idea what generates that either.  but can manually remove it.

[the remark this answers is not in the document]
Comment (smijar) [reply .gitignore:28]: confirmed final change is in to .gitignore

**Comment by smijar** [comment]
> LGTM after you confirm changes in `payloads/nginx/nginx/.gitignore`

confirmed that the change in .gitignore is final now.

**Pull request merged**